community/howto/120decibell/calculating_driver_current.md

```diff
@@ -6,22 +6,25 @@ nav_exclude: true
 
 # Calculating Driver Current Settings
 
-When using the 2208 / 2209 drivers, the voltage & current are set in software.  In Klipper, the motor currents have two settings: run and hold.  [However, it is no longer recommended][klipperTMCUpdate] to specify a `hold_current` for most motors. Klipper current settings are based on root-mean-squared (RMS) and **not** on peak current.  The spec sheets for most motors list the peak current capacity.
+When using the 2208 / 2209 drivers, the voltage & current are set in software.  In Klipper, the motor currents have two settings: run and hold.  [However, it is no longer recommended][klipperTMCUpdate] to specify a `hold_current` for most motors. Klipper current settings are based on root-mean-squared (RMS) and **not** on peak current.  Depending on the spec sheet the rated current is either peak or RMS.  If you are unsure, assume that it is peak.
```

Comment:
Would be great to add references to some other common drivers like the 5160!

Reply:
good idea. let me see what I can add.